jackal_control/config/control.yaml

```diff
@@ -1,5 +1,6 @@
 controller_manager:
   ros__parameters:
+    use_sim_time: False
```

Comment:
This can't be here for real robots.

Reply:
This line doesn't exist on my packages on the real robot (I don't commit and push on the real robot, which is likely why you see it here and not there. Though, I'm curious - if we set it to False, would this be sufficient?)